python/ccxt/pro/test/exchange/test_watch_balance.py

```diff
@@ -21,12 +21,16 @@ async def test_watch_balance(exchange, skipped_properties, code):
     ends = now + 15000
     while now < ends:
         response = None
+        success = True
```

Comment:
@Dan-krm can you pls revert these unwanted changes?

Reply:
It looks like these changes are already present in ccxt master so I committed them to my forked master, do you know why but it showed these changes uncommitted on my forked master after I synced?

Reply:
<img width="1040" height="856" alt="image" src="https://github.com/user-attachments/assets/ab8bef58-7872-43d8-962f-4d374dae9d39" />

Reply:
@Dan-krm your fork/branch might be out of sync, I added those changes a few days ago to the `ts` version of the files, if you don't have an updated `master` version locally those might appear

Reply:
I still haven't been able to sync these changes into my forked version but I opened a separate pull request without these unwanted changes